.github/workflows/ui-tests.yml

```diff
@@ -94,14 +100,14 @@ jobs:
         uses: actions/upload-artifact@v4
         if: ${{ failure() }}
         with:
-          name: ui-tests-ios-swiftui-${{matrix.device}}.xcresult
+          name: ui-tests-ios-swiftui.xcresult
           path: fastlane/test_results/ui-tests-ios-swiftui.xcresult
 
       - name: Archiving Raw Test Logs
         uses: actions/upload-artifact@v4
         if: ${{ failure() || cancelled() }}
         with:
-          name: ui-tests-ios-swiftui-raw-output-${{matrix.device}}
+          name: ui-tests-ios-swiftui-raw-output
```

Comment:
`m`: We should keep some value of the matrix in this name, otherwise when adding another matrix entry the upload will warn/fail with a "duplicate artifact name" error